c:\temp\maple-server-internal.log

```diff
@@ -0,0 +1,10 @@
+2021-03-23 17:59:11.8452 Info Auto loading assembly file: /home/andrew-gan/Documents/git/MapleServer2/MapleServer2/bin/Debug/net5.0/NLog.Extensions.Logging.dll
```

Comment:
What is this

Reply:
running the code typically outputs a log file in windows but the path is invalid on linux so it was created here instead.
Removed :p